presto-product-tests/bin/run_on_docker.sh

```diff
@@ -115,9 +115,8 @@ function terminate() {
 }
 
 ENVIRONMENT=$1
-
-if [[ "$ENVIRONMENT" != "singlenode" && "$ENVIRONMENT" != "multinode" && "$ENVIRONMENT" != "singlenode-kerberized" ]]; then
-   echo "Usage: run_on_docker.sh <singlenode|multinode|singlenode-kerberized> <product test args>"
+if [[ "$ENVIRONMENT" != "singlenode" && "$ENVIRONMENT" != "multinode" && "$ENVIRONMENT" != "singlenode-kerberized" && "$ENVIRONMENT" != "singlenode-kerberized-without-impersonation" ]]; then
+   echo "Usage: run_on_docker.sh <singlenode|multinode|singlenode-kerberized|singlenode-kerberized-without-impersonation> <product test args>"
```

Comment:
Instead of adding `ifs` here maybe betted do the `ls conf/docker` and check whether the specified environment is there.